web/benchmark-core/build.gradle.kts

```diff
@@ -9,8 +11,7 @@ kotlin {
         browser() {
             testTask {
                 useKarma {
-                    useChromeHeadless()
-                    //useFirefox()
+                    standardConf()
```

Comment:
this one actually was intentionally kept unchanged, as it uses nonStandard karma config to increase the timeouts.
with this change, the build fails due to timeouts in tests.

Reply:
Ah, I see. Will revert.

Reply:
This, by the way, can noticed if one will run `./gradlew clean build` ;)

Reply:
Reverted